app/assets/stylesheets/web/admin.scss

```diff
@@ -137,3 +137,17 @@
     text-transform: none;
   }
 }
+
+.admin-links-cols {
+  column-count: 2;
+  margin-bottom: 30px;
+
+  .col {
+    break-inside: avoid-column;
+  }
```

Comment:
This `.col` class looks like it isn't used. Perhaps it should replace the inline style on the project admin div?

Reply:
Yeah, you're right. Thanks for catching that. I cleaned that up in the latest commit.